Makefile

```diff
@@ -38,3 +38,23 @@ bblfsh-start:
 	docker exec style_analyzer_bblfshd bblfshctl driver install \
 		javascript docker://bblfsh/javascript-driver\:v1.2.0
 	(docker exec -it style_analyzer_bblfshd bblfshctl driver list | grep javascript) || (docker rm -f style_analyzer_bblfshd && false)
+
+REPORTS_DIR = reports
+REPORT_VERSION = untagged
+REPORT_DIR = $(REPORTS_DIR)/$(REPORT_VERSION)
+SMOKE_REPORT_DIR = $(REPORT_DIR)/js_smoke
+NOISY_REPORT_DIR = $(REPORT_DIR)/noisy
+QUALITY_REPORT_DIR = $(REPORT_DIR)/quality
+SMOKE_INIT = ./lookout/style/format/benchmarks/data/js_smoke_init.tar.xz
+.PHONY: release-report
+release-report:
```

Comment:
```
$(SMOKE_REPORT_DIR) $(NOISY_REPORT_DIR) $(QUALITY_REPORT_DIR):
    mkdir -p $@
report_smoke: $(SMOKE_REPORT_DIR)
    # .. smoke code
noisy_report: $(NOISY_REPORT_DIR)
    # noisy code
...etc...
release-report: report_smoke noisy_report quality_report
```

Reply:
I hope I get it right.
So no `.PHONY` for any of this commands?

Reply:
`.PHONY` is required for these commands.

Also should all be put in one single line to avoid bloating the Makefile

Reply:
sorry I do not get the last one.

Reply:
basically, transforming all the lines
```
.PHONY: some-target
...
.PHONY: some-other-target
```
into
```
.PHONY: some-target some-other-target
```